unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/WearableGridController.cs

```diff
@@ -0,0 +1,162 @@
+            string ownUserId = ownUserProfile.userId;
+
+            try
+            {
+                // TODO: instead of requesting owned wearables, we should request all the wearables with the current filters & sorting
+                (IReadOnlyList<WearableItem> wearables, int totalAmount) = await wearablesCatalogService.RequestOwnedWearablesAsync(
+                    ownUserId,
+
+                    // page is not zero based
+                    page + 1,
+                    PAGE_SIZE, true, cancellationToken);
+
+                var wearableModels = wearables.Select(wearable =>
+                {
+                    if (!Enum.TryParse(wearable.rarity, true, out NftRarity rarity))
+                    {
+                        rarity = NftRarity.Common;
+                        Debug.LogError($"Could not parse the rarity of the wearable: {wearable.rarity}. Fallback to common..");
+                    }
+
+                    return new WearableGridItemModel
+                    {
+                        WearableId = wearable.id,
+                        Rarity = rarity,
+                        ImageUrl = wearable.ComposeThumbnailUrl(),
+                        IsEquipped = ownUserProfile.HasEquipped(wearable.id),
+
+                        // TODO: make the new state work
+                        IsNew = false,
+                        IsSelected = false,
+                    };
+                });
+
+                view.SetWearablePages(page, (totalAmount / PAGE_SIZE) + 1);
+
+                // TODO: mark the wearables to be disposed if no references left
+                view.ClearWearables();
+                view.ShowWearables(wearableModels);
+
+                return totalAmount;
+            }
+            catch (Exception e) { Debug.LogException(e); }
+
+            return 0;
+        }
+
+        private void HandleWearableSelected(WearableGridItemModel wearableGridItem)
+        {
+            view.ClearWearableSelection();
+            view.SelectWearable(wearableGridItem.WearableId);
+        }
+
+        private void HandleWearableUnequipped(WearableGridItemModel wearableGridItem)
+        {
+            throw new NotImplementedException();
+        }
+
+        private void HandleWearableEquipped(WearableGridItemModel wearableGridItem)
+        {
+            throw new NotImplementedException();
+        }
+
+        private void FilterWearablesFromBreadcrumb(string referencePath)
+        {
+            if (referencePath.StartsWith("all://"))
+            {
+                requestWearablesCancellationToken = requestWearablesCancellationToken.SafeRestart();
+                ShowWearablesAndItsFilteringPath(0, requestWearablesCancellationToken.Token).Forget();
+            }
+            else if (referencePath.StartsWith("name://")) { throw new NotImplementedException(); }
+            else if (referencePath.StartsWith("category://")) { throw new NotImplementedException(); }
```

Comment:
I have already seen this kind of hardcoded texts (`all://`, `name://`, `category://`...) several times in different parts of the code. Could we extract them into constants to make them more accessible and easier to maintain?

Reply:
Extracted to constants